wordpress-mobile/WordPress-Android
# Fix rtl layouts in My Sites section

Partially fixes #2278 - Fixes RTL layouts on my sites screens

Some Views are not properly positioned in the RTL language mode.

To test: 
1. Check layouts of the My Sites + sub-screens
2. Change your locale to a RTL language (eg. Hebrew) or force RTL layouts in the developer options.
3. Check the layouts, see step 1

Doesn't fix the ViewPager swipe direction in the RTL mode.
